<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModulePannel</class>
 <widget class="QFrame" name="ModulePannel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0795909, y1:0.119, x2:0.983, y2:0.998455, stop:0 rgba(117, 117, 117, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(0, 0, 0);</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::Box</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Plain</enum>
  </property>
  <widget class="QPushButton" name="pushButton_ModuleExit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.130864, y1:0.119, x2:0.779, y2:0.7435, stop:0 rgba(156, 156, 156, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(126, 126, 126);</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ModuleOpen">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>80</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0513182, y1:0.062, x2:0.955, y2:0.942364, stop:0 rgba(156, 156, 156, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(126, 126, 126);</string>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ModuleConnections">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.0513182, y1:0.062, x2:0.955, y2:0.942364, stop:0 rgba(156, 156, 156, 255), stop:1 rgba(255, 255, 255, 255));
border-color: rgb(126, 126, 126);</string>
   </property>
   <property name="text">
    <string>Connections</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_ModuleMIDIin">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>91</width>
     <height>25</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="LinearGradientPattern">
        <gradient startx="0.079590900000000" starty="0.119000000000000" endx="0.983000000000000" endy="0.998455000000000" type="LinearGradient" spread="PadSpread" coordinatemode="ObjectBoundingMode">
         <gradientstop position="0.000000000000000">
          <color alpha="255">
           <red>117</red>
           <green>117</green>
           <blue>117</blue>
          </color>
         </gradientstop>
         <gradientstop position="1.000000000000000">
          <color alpha="255">
           <red>255</red>
           <green>255</green>
           <blue>255</blue>
          </color>
         </gradientstop>
        </gradient>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
background-color: qlineargradient(spread:pad, x1:0.0795909, y1:0.119, x2:0.983, y2:0.998455, stop:0 rgba(117, 117, 117, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string>MIDI IN</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="tristate">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>5</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Module Name</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
